src/core/agent/services/multiSigService.ts

```diff
@@ -716,6 +722,149 @@ class MultiSigService extends AgentService {
     }
     return true;
   }
+
+  async getMultisigParticipants(multisigSignifyName: string) {
+    const members = await this.props.signifyClient
+      .identifiers()
+      .members(multisigSignifyName);
+    const multisigMembers = members["signing"];
+    const allMultisigMemberIdentifiers =
+      await this.identifierStorage.getMultisigMemebersMetadata();
+    const owningMember = multisigMembers.find((member: any) =>
```

Comment:
Can we not search with SQL for the IDs of the members? Rather than cross checking all of them in JS